Sources/Utility/StringExtensions.swift

```diff
@@ -70,4 +70,11 @@ extension String {
 
         return String(cc)
     }
+    
+    /// Splits string with any character in passed-in string of delimiters
+    public func splitWithCharactersInString(delimiters: String) -> [String] {
+        let stops = Set(delimiters.characters)
+        return self.characters.split { stops.contains($0) }.map(String.init)
```

Comment:
But thanks for pointing it out, I always forget you can just pass the function itself in. :+1: I only tried this after you commented here, unfortunately couldn't get it to work though.